src/main/java/de/qabel/desktop/ui/feedback/FeedbackController.java

```diff
@@ -28,38 +28,50 @@
 
     @Inject
     private CrashReportHandler reportHandler;
+    private ResourceBundle resourceBundle;
+    String infoMessage;
+    String titleBar;
```

Comment:
titleBar is a bit misleading. sounds like a UI Object. may call it infoTitle or something...